packages/generator/src/sdk-metadata/sdk-metadata-types.ts

```diff
@@ -68,13 +68,13 @@ export interface GenerationAndUsage {
   links: Links;
 }
 
-type UrlString = string;
+export type UrlString = string;
```

Comment:
Maybe I have missed something here, but why?

Reply:
In the type everything is a string. I wanted to get some means to distinguis a URL, from a time from a multi line code block. @marikaner does the `JSON.stringify()` call the `toString()` on each attribute so that one could use the real type like URL, datatime, codeblock and implement a `toString()`?